Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -170,6 +173,24 @@ public void setupDriveCommand(
     }
     
     @Inject
+    public void setupVisionCommands(
+            OperatorInterface oi,
+            RotateRobotToBoilerCommand rotateCommand,
+            DriveToBoilerWithVisionCommandGroup driveToBoilerCommand,
+            SetRobotHeadingCommand setHeadingCommand,
+            RotateToHeadingCommand rotateToHeadingCommand
+    )   {
+        oi.leftButtons.getifAvailable(7).whileHeld(rotateCommand);
+        oi.leftButtons.getifAvailable(8).whileHeld(driveToBoilerCommand);
+        
+        setHeadingCommand.setHeadingToApply(248);
```

Comment:
I see it's "Angle of line parallel to boiler". I'd suggest putting that as a static variable in the pose subsystem or the vision subsystem.